### PR TITLE
Implemented dynamic frameworks gathering from available on machine

### PR DESCRIPTION
**Pull request by xperiandri**
Also added basic Editorconfig.
Set up more rules as you prefer.

### CODE REVIEW

**Comment by xperiandri** [comment]
https://www.mztools.com/articles/2015/MZ2015017.aspx

**Comment by 304NotModified** [comment]
Cool!


I will try to test this, if it all works well, I will merge it soon!

**Comment by 304NotModified** [comment]
maybe because i'm debugging, but I get this issue when pressing "migrate"

image

**Comment by 304NotModified** [comment]
LOL this is a long list:

image

**Comment by 304NotModified** [comment]
Cool this works also for SDK style projects. Unfortunaly I cannot switch back to netcoreapp3.1 (or any netcoreapp). It's not in the list:

image

**Comment by 304NotModified** [comment]
It this correct? ".NETCore,Version=v4.5"?

image

image

**Comment by 304NotModified** [comment]
FYI,

Examples of correct monikers.

```
  ".NET Standard,Version=v2.1"
  ".NETStandard,Version=v2.1",
  ".NET Core,Version=v2.1",
  ".NET Core,Version=v3.1",
  ".NET Core,Version=v3.1",
  ".NETFramework,Version=v2.0"

```
Funny that space is optional? (but display name is then also without space, see screenshot below)

unfortunaly this doesn't give a displayname:

```
"netcoreapp3.1",
"netstandard2.1",
"net472"
```
screenshot:

image

**Comment by xperiandri** [comment]
It is just a matter of presentation.
I would prefer to rewrite UI in WPF then this presentation stuff would be pretty simple.

**Comment by 304NotModified** [comment]
This isn't solved with a UI rewrite? With this method (and also of course the current source), we can't select .NET Core / .NET Standard

Maybe it's an idea to combine this and the old approach? So bring back the Frameworks.xml , but list there the names  like

````
 ".NET Standard,Version=v2.1"
  ".NET Core,Version=v2.1",
  ".NET Core,Version=v3.1",
  ".NET Core,Version=v3.1",
  ".NETFramework,Version=v2.0"

```

I'ts not dynamic, but we win that it's easier to maintain.

What do you think?

**Comment by xperiandri** [comment]
I've rebase my changes on top of latest

File: Src/TargetFrameworkMigrator/Models/ProjectModel.cs [hunk 1]
```diff
@@ -1,71 +0,0 @@
-﻿using System.ComponentModel;
```
Comment (304NotModified) [inline Src/TargetFrameworkMigrator/Models/ProjectModel.cs]: any idea why this one is marked as removed and added?

**Comment by 304NotModified** [comment]
What about this? https://github.com/TargetFrameworkMigrator/TargetFrameworkMigrator/pull/41#issuecomment-567250240


Unfortunately there is something broken: see https://github.com/TargetFrameworkMigrator/TargetFrameworkMigrator/issues/83. Any idea what could be the issue? it's a showstopper

**Comment by 304NotModified** [comment]
Closing this due to inactivity. Please let us know if this still an issue and please provide the requested info -
 thanks!

**Comment by xperiandri** [comment]
So why not merge this PR?

**Pull request closed without merging**